### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by LionShareZoki**
Create initial readme on how to start a project locally

### CODE REVIEW

File: frontend/README.md [hunk 1]
```diff
@@ -1,27 +1,24 @@
-# React + TypeScript + Vite
+# Vite React TypeScript Starter
```
Comment (dariomrk) [inline frontend/README.md:1]: Could we change this to something like `# Taskify Frontend`?

**Review by dariomrk — CHANGES_REQUESTED** [review]
The `README.md` should contain these sections:
- Heading (title)
- Description (what is this all about)
- Getting started (dev environment setup instructions)
  The dev setup instructions should contain the minimal amount of information to set up the repository for local development.
  The setup instructions should be logically ordered. It is very helpful if terminal commands are clearly formatted like `so`.

File: frontend/README.md (hunk 1 above)
Comment (LionShareZoki) [reply frontend/README.md:1]: Yep

File: frontend/README.md [hunk 2]
```diff
@@ -1,27 +1,24 @@
...
+3. **Install dependencies:**
 
-- Replace `plugin:@typescript-eslint/recommended` to `plugin:@typescript-eslint/recommended-type-checked` or `plugin:@typescript-eslint/strict-type-checked`
-- Optionally add `plugin:@typescript-eslint/stylistic-type-checked`
-- Install [eslint-plugin-react](https://github.com/jsx-eslint/eslint-plugin-react) and add `plugin:react/recommended` & `plugin:react/jsx-runtime` to the `extends` list
+   `npm install`
+
+4. **Start the development server:**
+
+   `npm run dev`
```
[the remark this answers is not in the document]
Comment (LionShareZoki) [reply frontend/README.md:22]: If `git clone` will be in the root repo I will assume to make instructions after the whole project is cloned.

**Comment by LionShareZoki** [comment]
I have made a new commit. Can you see it now? Is it pushed correctly since I have committed it directly on GitHub?

**Pull request merged**